libs/brig-types/test/unit/Test/Brig/Types/Common.hs

```diff
@@ -86,13 +86,27 @@ instance Show TeamMemberDeleteData where
   show a = "(TeamMemberDeleteData " <> show (a ^. tmdAuthPassword) <> ")"
 
 instance Arbitrary SSOStatus where
-  arbitrary = Test.Tasty.QuickCheck.elements [minBound ..]
```

Comment:
This will conflict with https://github.com/wireapp/wire-server/pull/1086 - @fisx can we reach a consensus here? :)

Reply:
Somehow I missed [this comment](https://github.com/wireapp/wire-server/pull/1086/files#r419934139) from @fisx yesterday. I think we can have this discussion in that PR. In all this PR doesn't really add/remove anything from galley/brig-types. So I will revert my refactoring